### ISSUE TITLE
[CRASH] invalid .releaseconfig.json file

### ISSUE THREAD

**Issue by mcm1957** [ioBroker/ioBroker.repochecker#283]
```

checkCode [E5xx]
Download https://github.com/SKB-CGN/ioBroker.energiefluss/archive/main.zip
main.zip 2859379 bytes
context[/.releaseconfig.json: {
    // Boolean, number, string as is:
    "dry": false,
    // Arrays:
    "plugins": [
        "iobroker",
        "license"
    ],
    // Objects
    "exec": {
        "before_commit": "Doing commit!"
    }
}
GLOBAL ERROR: SyntaxError: Expected property name or '}' in JSON at position 7, {}
```

**Comment by mcm1957** [comment]
see https://github.com/Grizzelbee/ioBroker.mielecloudservice/blob/master/.releaseconfig.json

**Comment by mcm1957** [comment]
fixed 3.0.4
